SPECS-EXTENDED/hdf/hdf.spec

```diff
@@ -0,0 +1,578 @@
+# No more Java on i686
+%ifarch %{java_arches}
+%bcond_without java
+%else
+%bcond_with java
+%endif
+Summary:        A general purpose library and file format for storing scientific data
+Name:           hdf
+Version:        4.2.15
+Release:        13%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
```

Comment:
Just wondering, we used to have a 'Group' tag here. Did we stop adding it?

Reply:
Nope, its just `best-effort` IMO.
Feel free to add it if you know what the `Group` is